### DISCUSSION TITLE
HTTP Toolkit

### DISCUSSION THREAD

**Discussion by ivan-curkovic** [docker/extension-ideas#140] [Ideas]
Integrate HTTP Toolkit into Docker to intercept, debug and mock HTTP requests. 

Website: https://httptoolkit.com/
Github: https://github.com/httptoolkit

---

_Does this affect you too? Click the :smiley: icon in the top right and add a :+1: on issues to help decide what Docker Extensions to build, or [go vote on the most popular ideas so far](https://github.com/docker/extension-ideas/discussions?discussions_q=sort%3Atop)._

**Comment by pimterry** [comment]
There's a better landing page that's fully focused on Docker, with a demo of the existing Docker terminal integration, at https://httptoolkit.com/docker/.

The docs might also be interesting: https://httptoolkit.com/docs/guides/docker/. There's functional info there but also a full breakdown of [how the internals work](https://httptoolkit.com/docs/guides/docker/#the-technical-details).
